### ISSUE TITLE
Question about GPU

### ISSUE THREAD

**Issue by PinRueiCCU** [littlepure2333/MindBridge#9]
Thank you for your outstanding project, this is a truly impressive paper.

In your Acknowledgement section, you mentioned:
`Our appreciation also extends to Accelerate and DeepSpeed for simplifying the process of efficient multi-GPU training, enabling us to train on the 24GB vRAM GPU, NVIDIA A5000.`

I would like to ask if it is possible to train MindBridge using a single GPU?

**Comment by littlepure2333** [comment]
Hi, thank you for your interest in our work! Yes, you can train the model using a single GPU by simply running the command `python main.py`. For example, in the script `scripts/train_single.sh`, you can modify it as follows:

```bash
python main.py \
--model_name $model_name --subj_list $subj \
--num_epochs $num_epochs --batch_size $batch_size --val_batch_size $val_batch_size \
--h_size 2048 --n_blocks 4 --pool_type max --pool_num 8192 \
--mse_mult $mse_mult \
--eval_interval 10 --ckpt_interval 10 \
--max_lr 3e-4 --num_workers 4
```

**Comment by PinRueiCCU** [comment]
Hello, thanks for your previous instructions on running ``train_single.sh`` on a single GPU — I was able to train successfully following your steps. However, when I attempted to modify ``train_bridge.sh`` for single GPU training, I encountered two issues:
1.	Even after reducing the number of processes, I ran into an out-of-memory error, which suggests that the multi-process configuration may be causing excessive memory usage on a single GPU.
2.	To mitigate the memory issue, I tried adding a gradient accumulation parameter (e.g., ``--gradient_accumulation_steps 2``) to maintain an effective batch size while using smaller mini-batches. Unfortunately, the code did not recognize this argument, indicating that gradient accumulation isn’t currently implemented.

Could you please clarify whether train_bridge.sh can be trained on a single GPU? If so, could you advise on the recommended modifications—such as reducing the number of processes, adjusting the batch size, or implementing gradient accumulation—to run the training successfully under a single GPU setup?

Thank you for your assistance!
